### DISCUSSION TITLE
Easy way to contact you for attribution

### DISCUSSION THREAD

**Discussion by Signorlimone** [civitai/civitai#50] [Ideas]
Hi, I'm Signorlimone. Great work you've done with your site, I like the UI, works really well. I've searched a way to contact you for attribution of the model I trained (until now I found 3 model wich are: StylizR, Clazy and Caricaturizer). I already made an account on Civitai. Please, let me know

**Comment by JustMaier** [comment]
Hey @Signorlimone I've transferred the ownership of those over to you.

I agree with you, it'd be best if there was a way for creators to claim models that we've imported. I'll talk with the team and look at adding a way to claim ownership. I'll post here when that's live!

**Reply by Signorlimone** [reply]
thank you very much

**Comment by JustMaier** [comment]
We added a way for creators to claim models that were imported here:
![image](https://user-images.githubusercontent.com/607609/211396140-838e3d02-f795-4efa-a21c-571976d678e1.png)

Hopefully this makes it easier for people to request ownership of models uploaded by Civitai or other users.
